docs/_docs/Instructors/02-Teachers-How-to-build-Learning-Units.md

```diff
@@ -28,6 +28,8 @@ out more information [here](../../Starters Academy (LDSSA)/01-Starters-Academy-(
 
```

Comment:
There's a typo on line 20, probably from before

"Solkution" instead of "Solution"

Reply:
Fixed